src/dev/storybook/aliases.ts

```diff
@@ -67,7 +67,7 @@ export const storybookAliases = {
   presentation: 'src/platform/plugins/shared/presentation_util/storybook',
   random_sampling: 'x-pack/platform/packages/private/kbn-random-sampling/.storybook',
   esql_editor: 'src/platform/packages/private/kbn-esql-editor/.storybook',
-  security_solution: 'x-pack/solutions/security/plugins/security_solution/.storybook',
+  // security_solution: 'x-pack/solutions/security/plugins/security_solution/.storybook',
```

Comment:
maybe it's preferable to skip with a link to an issue, or skip in the iteration @ https://github.com/elastic/kibana/blob/16f69b7b2dd6b579c6ab592563a187cb7c1f10d8/.buildkite/scripts/steps/storybooks/build_and_upload.ts#L41

Reply:
I've added a link for the issue 👍